app-misc/diff-so-fancy/diff_so_fancy-1.4.3.recipe

```diff
@@ -0,0 +1,32 @@
+SUMMARY="Pretty diff generator"
+DESCRIPTION="diff-so-fancy strives to make your diffs human readable instead of machine readable. This helps improve code quality and helps you spot defects faster.
+
+Configure git to use diff-so-fancy for all diff output:
+
+git config --global core.pager \"diff-so-fancy | less --tabs=4 -RFX\"
+git config --global interactive.diffFilter \"diff-so-fancy --patch\""
+HOMEPAGE="https://github.com/so-fancy/diff-so-fancy"
+COPYRIGHT="2016 So Fancy team"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/so-fancy/diff-so-fancy/releases/download/v$portVersion/diff-so-fancy#noarchive"
```

Comment:
It might be better to download the whole release using the standard
```
https://github.com/so-fancy/diff-so-fancy/archive/refs/tags/v$portVersion.tar.gz
```

so that you can copy the README.md (and the other markdown files?) to `$docDir`

Reply:
That's annoying.  In theory you could add an extra SOURCE_URI to get the markdown files but I don't know if it's critical.